Makefile

```diff
@@ -40,3 +47,27 @@ markdownlint:
 		npx --no -p markdownlint-cli markdownlint -c .markdownlint.yaml $$f \
 			|| exit 1; \
 	done
+
+
+GENDIR := gen
+# Find all .proto files.
+PROTO_FILES := $(wildcard proto/*.proto)
+
+PROTO_GEN_GO_DIR ?= $(GENDIR)/go
+
+gen-proto: gen-go
+.PHONY: gen-proto
+
+# Generate Protobuf Go files.
+.PHONY: gen-go
+gen-go:
```

Comment:
I tried running this command, but the docker image doesn't exist for my platform (linux/arm64/v8)

Is the purpose of this rule to check that the go files are correctly generated?

Reply:
Yes, that's the purpose. It is using the same docker image as https://github.com/open-telemetry/opentelemetry-proto